### PR TITLE
docs: improve font styling✔

### PR DESCRIPTION
**Pull request by thetejasmagade**
1. Fixed font-family
2. improved font styling of description on index page
3. improved font styling of documentation pages
4. improved font styling of header

### CODE REVIEW

**Review by qmhc — CHANGES_REQUESTED** [review]
Have some unexpected changes, for those subjective like `font-size`, `line-height` and `letter-spacing`, I think these should not be changed.

File: docs/common/header.vue [hunk 1]
```diff
@@ -224,6 +224,7 @@ function formatComponentName(name: string) {
   align-items: center;
   width: 100%;
   height: var(--header-height);
+  font-weight: 500;
```
[the remark this answers is not in the document]
Comment (thetejasmagade) [reply docs/common/header.vue:227]: It's improving style of header links

**Without this**
Screenshot 2022-08-20 144106

**With This**
Screenshot 2022-08-20 190256

File: docs/views/homepage.vue [hunk 2]
```diff
@@ -68,8 +68,10 @@ function toRepository() {
 
   &__description {
     margin-bottom: 1rem;
-    font-size: 1.4rem;
+    font-size: 1.3rem;
     font-weight: 300;
+    font-weight: 400;
```
[the remark this answers is not in the document]
Comment (thetejasmagade) [reply docs/views/homepage.vue:73]: Yes

File: docs/common/header.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (qmhc) [reply docs/common/header.vue:227]: I tend to use thin font style, it is better on different devices.

[the remark this answers is not in the document]
Comment (thetejasmagade) [reply docs/common/header.vue:227]: Should, I remove it?

[the remark this answers is not in the document]
Comment (qmhc) [reply docs/common/header.vue:227]: yes.

[the remark this answers is not in the document]
Comment (thetejasmagade) [reply docs/common/header.vue]: Removed

File: docs/views/homepage.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (qmhc) [reply docs/views/homepage.vue:72]: Here I mean it shouldn't change `font-weight` to `400`. 😂

File: docs/common/markdown.vue [hunk 3]
```diff
@@ -49,6 +49,9 @@ watch(isMounted, value => {
 
   p {
     margin: 14px 0;
+    font-size: 17px;
```
Comment (qmhc) [inline docs/common/markdown.vue:52]: I mean should not add these styles, so you need to remove it, includes `font-size`, `line-height` and `letter-spacing`.

**Review by qmhc — CHANGES_REQUESTED** [review]

File: docs/common/markdown.vue (hunk 3 above)
Comment (thetejasmagade) [reply docs/common/markdown.vue:52]: ok

File: docs/views/homepage.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (thetejasmagade) [reply docs/views/homepage.vue:72]: So, there are two proprties of font-weight. Which do you want? 😊

[the remark this answers is not in the document]
Comment (qmhc) [reply docs/views/homepage.vue:72]: It's `300`.

[the remark this answers is not in the document]
Comment (thetejasmagade) [reply docs/views/homepage.vue]: Ok! done

**Review by qmhc — APPROVED** [review]
LGTM, thanks for your works.

**Comment by thetejasmagade** [comment]
❤️

**Pull request merged**